2-mandatory/5-writing-tests.js

```diff
@@ -35,41 +35,65 @@ function convertScoreToGrade(score) {
   passes.
 */
 test("a score of 83 is grade A", () => {
-  expect(convertScoreToGrade(83), "Z");
+  expect(convertScoreToGrade(83)).toEqual("A");
 });
 
 /*
   The rest of the tests have comments describing what to test and you need to
   write a matching test
 */
 
-test.skip("a score of 71 is grade B", () => {
+test("a score of 71 is grade B", () => {
+  expect(convertScoreToGrade(71)).toEqual("B");
   /* Remove the .skip above, then write the test body. */
 });
-/*
-  Write a test that checks a score of 68 is grade C
+
+
+ /* Write a test that checks a score of 68 is grade C
 */
+test("a score of 68 is grade C", () => {
+  expect(convertScoreToGrade(68)).toEqual("C");
+});
 
+  
 /*
   Write a test that checks a score of 55 is grade D
 */
+test("a score of 55 is grade D", () => {
+  expect(convertScoreToGrade(55)).toEqual("D");
+});
+ 
 
 /*
   Write a test that checks a score of 68 is grade C
 */
+test("a score of 68 is grade C", () => {
+  expect(convertScoreToGrade(68)).toEqual("C");
+});
 
 /*
   Write a test that checks a score of 55 is grade D
 */
+test("a score of 68 is grade D", () => {
+  expect(convertScoreToGrade(55)).toEqual("D");
+});
 
 /*
   Write a test that checks a score of 49 is grade E
 */
+test("a score of 68 is grade E", () => {
+  expect(convertScoreToGrade(49)).toEqual("E");
+});
 
 /*
   Write a test that checks a score of 30 is grade E
 */
-
+test("a score of 68 is grade E", () => {
+  expect(convertScoreToGrade(30)).toEqual("E");
+});
 /*
   Write a test that checks a score of 70 is grade B
 */
+test("a score of 68 is grade B", () => {
+  expect(convertScoreToGrade(70)).toEqual("B");
+});
```

Comment:
❓ Have you checked the names of these tests  - it looks like you may have copy and pasted without updating